### PR TITLE
Fixes #8 : Added Initial Landing Content

### LINKED ISSUE #8: Need to add landing content
The website has template content, should be updated.
Fossasia related content should be added.

Working on it

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #8 
Replaced irrelevant template content with Fossasia related content.
Added Logo to navbar
[Preview Link](https://saarthakchats.github.io/newsite.fossasia.org/)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -648,15 +648,15 @@ <h6 class="title">Shopping Cart</h6>
...
-                                        <br /> It's your design toolkit.</h1>
-                                    <h6 class="uppercase mb16">A performance focussed template.</h6>
-                                    <p class="lead mb40">
-                                        Build beautiful, contemporary sites in just moments
-                                        <br />with Foundry and Variant Page Builder.
-                                    </p>
+                                    <h1 class="mb40 mb-xs-16 large" style="color:white;">We are
+                                        <br/>Fossasia</h1>
+                                    <h6 class="uppercase mb16" style="color:white;">Creating collaborative open-source technologies.</h6>
+                                    <p class="lead mb40" style="color:white;">
```
Comment (abishekvashok) [inline index.html]: Please don't use inline styles.

**Comment by saarthakchats** [comment]
@abishekvashok 
take a look now 💯

**Comment by YashKumarVerma** [comment]
Do we really need the .ds store files?

**Comment by saarthakchats** [comment]
I am currently having some troubles squashing commits, I'll do it from the next PR

**Comment by saarthakchats** [comment]
@abishekvashok please merge

File: index.html [hunk 2]
```diff
@@ -657,6 +668,7 @@ <h6 class="uppercase mb16">A performance focussed template.</h6>
                                         Build beautiful, contemporary sites in just moments
                                         <br />with Foundry and Variant Page Builder.
                                     </p>
+>>>>>>> parent of 20f7613... Added Initial Landing Content
```
Comment (abishekvashok) [inline index.html]: We can't have effects of a conflict.

**Comment by saarthakchats** [comment]
I don't think it conflicts now
@abishekvashok 
Please check

**Comment by saarthakchats** [comment]
I'll make a new PR

**Pull request closed without merging**